docker/gcb/oss-fuzz-coverage.yaml

```diff
@@ -28,15 +28,15 @@ steps:
     # Use two tags so that the image builds properly and we can push it to the
     # correct location.
     '--tag',
-    'gcr.io/fuzzbench/oss-fuzz/builders/coverage/${_BENCHMARK}-intermediate',
+    'gcr.io/fuzzbench/builders/coverage/${_BENCHMARK}-intermediate',
 
     '--tag',
-    '${_REPO}/oss-fuzz/builders/coverage/${_BENCHMARK}-intermediate',
+    '${_REPO}/oss-fuzz/coverage/${_BENCHMARK}-intermediate',
```

Comment:
`${_REPO}/builders/coverage/${_BENCHMARK}-intermediate`. 
Otherwise, the coverage build of OSS-Fuzz benchmarks fails in Step 3.

Reply:
Thanks, proposed PR in https://github.com/google/fuzzbench/pull/297